core-common/src/main/java/org/apache/kylin/common/KylinConfigBase.java

```diff
@@ -933,7 +933,7 @@ public String getHiveDependencyFilterList() {
                 + "[^,]*hive-metastore[^,]*?\\.jar" + "|" + "[^,]*hive-hcatalog-core[^,]*?\\.jar");
     }
 
-    public boolean isMailEnabled() {
+    public boolean isNotifyEnabled() {
```

Comment:
IMO, `isNotificationEnabled` looks better?

Reply:
notification is a noun, notify is a verb， I think it’s better to use verbs here， What do you think